test/string.check.spec.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const expect = require('chai').expect;
+const sinon = require('sinon');
+const config = require('./fixtures/config/stringCheckFixture').checks[0];
+const proxyquire = require('proxyquire').noCallThru().noPreserveCache();
+
+describe('String Checker', function(){
+
+	let StringCheck;
+	let mockFetch;
+
+	function setup(status, body){
+		mockFetch = sinon.stub().returns(Promise.resolve({
+			status: 200,
+			ok: status < 300,
+			text: () => Promise.resolve(body)
+		}));
+		StringCheck = proxyquire('../src/checks/string.check', {'node-fetch':mockFetch});
+		return new StringCheck(config);
+	}
+
+	it('Should be ok if the url returns the expected value', function(done){
+		const check = setup(200, 'OK');
+		check.start();
```

Comment:
Shame you can't access the `tick` promise, it would make these tests much cleaner. As it is if they fail then Mocha will give bad errors (it'll only provide timeout errors from never hitting `done();`)